LiteCore/Support/Channel.cc

```diff
@@ -10,9 +10,11 @@
 // the file licenses/APL2.txt.
 //
 
-#include "Channel.hh"
-#include "Error.hh"
+#if 0
```

Comment:
Why to keep it?

Reply:
I am universally in favor of not committing commented or ifdef'd code.  If it is needed again, git has it in its history.  Perhaps the next part of this PR can remove it.